ic-agent/src/agent/agent_config.rs

```diff
@@ -23,6 +25,7 @@ impl Default for AgentConfig {
             identity: Arc::new(AnonymousIdentity {}),
             ingress_expiry: None,
             transport: None,
+            verify_query_signatures: true,
```

Comment:
For verify_query_signatures we set the default to true in the config.
For ingress_expiry, we set the default to 300 seconds in Agent::new.

Just wondering why it's done differently.

Reply:
We don't guarantee that ingress_expiry's default has to be representable as a single value. For example, we could restrict minute rounding behavior to only apply when ingress_expiry is not explicitly defined. But verifying query signatures is a binary state - either it happens or it doesn't.

Reply:
#481